test/integration/grpcProvider.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { CHAIN_ID } from '../../src'
+import { account, grpcProvider, publicProvider } from './setup'
+import { StakerEntry } from '../../src/generated/grpc/massa/model/v1/staker_pb'
+import {
+  Slot as grpcSlot,
+  SlotRange,
+} from '../../src/generated/grpc/massa/model/v1/slot_pb'
+import {
+  AddressBalanceCandidate,
+  AddressExistsCandidate,
+  ExecutionQueryRequestItem,
+} from '../../src/generated/grpc/public_pb'
+
+let blockId: string
+let currentPeriod: number
+
+let minimalFees: bigint
+describe('Provider GRPC tests', () => {
+  beforeAll(async () => {
+    const networkInfos = await publicProvider.networkInfos()
+    minimalFees = networkInfos.minimalFee
+  })
+  test('getNodeStatus', async () => {
+    const status = await grpcProvider.getNodeStatus()
+    expect(status.config).toBeDefined()
+    expect(status.currentCycle).toBeDefined()
+    expect(status.currentTime).toBeDefined()
+    expect(status.currentCycleTime).toBeDefined()
+    expect(status.nextCycleTime).toBeDefined()
+    expect(status.nodeId).toBeDefined()
+    expect(status.version).toBeDefined()
+    expect(status.chainId).toBe(Number(CHAIN_ID.Buildnet))
+    expect(status.minimalFees).toBe(minimalFees)
+  })
+
+  test('getTransactionsThroughput', async () => {
+    const throughput = await grpcProvider.getTransactionsThroughput()
+    expect(throughput).toBeDefined()
+  })
+
+  test('getStakers', async () => {
+    const stakers: Array<StakerEntry> = await grpcProvider.getStakers()
+    expect(stakers).toBeDefined()
+    expect(stakers[0].getRolls()).toBeGreaterThan(0)
+  })
+
+  test('networkInfos', async () => {
+    const resp = await grpcProvider.networkInfos()
+    expect(resp.name).toBeDefined()
+    expect(resp.chainId).toBeDefined()
+    expect(resp.url).toBeDefined()
```

Comment:
can you go a little bit further on testing?
You can retrieve theses values from json rpc and compare